rocky/tests/test_boefjes_tasks.py

```diff
@@ -91,6 +94,34 @@ def test_reschedule_task(rf, client_member, mocker, task):
     )
 
 
+def test_reschedule_task_from_other_org(rf, client_member, client_member_b, mocker, task):
+    mock_scheduler_client = mocker.patch("rocky.views.tasks.client")
+    mock_scheduler_client.get_task_details.return_value = task
+    mocker.patch("tools.view_helpers.client.session")
+    mocker.MagicMock()
+
+    request = setup_request(
+        rf.post(
+            f"/en/{client_member.organization.code}/tasks/boefjes/?task_id={task.id}",
+            data={"action": "reschedule_task"},
+        ),
```

Comment:
@underdarknl I was just thinking: should this view within this context yield a permission error instead, or is that too much information disclosure?

Reply:
@ammar92 Clients can not start the task (for example another redteam member) but can view and download task details.

Reply:
We should make sure there is no difference between asking for a nonexistent Job, OOI, task or organisation to be rescheduled, and for one of those that does exists but is not available to you as a user.